bitnami/grafana-operator/Chart.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: v2
```

Comment:
We should be using `v1` because support for Helm v2 is also provided

Reply:
Yeah I need to work on this ;)